x-pack/plugins/ml/server/routes/schemas/data_visualizer_schema.ts

```diff
@@ -11,6 +11,14 @@ export const indexPatternTitleSchema = schema.object({
   indexPatternTitle: schema.string(),
 });
 
+export const dataVisualizerFieldHistogramsSchema = schema.object({
+  /** Query to match documents in the index. */
+  query: schema.any(),
+  fields: schema.arrayOf(schema.any()),
```

Comment:
Nit - can you add some sort of comment for `fields` too.

Reply:
Updated in b794e97.